src/AddTodoForm.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+// This code is a functional component called AddTodoForm that takes a single prop called props.
```

Comment:
Great job documenting your code by writing comments. Typically comments are brief descriptions of what a line or lines of code do, especially when it might be a little confusing look at later. In this case, it may actually take longer to read the comments than see what's going on in the code!